app/views/ImporterAddressConfirmationView.scala.html

```diff
@@ -0,0 +1,87 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import controllers.routes._
+@import models.Mode
+
+@this(
+        main_template: MainTemplate,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+        govukSelect: GovukSelect,
+        link: components.link
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@pageHint = {
+    <h2 class="govuk-heading-m" style="margin-bottom: 0.3em">
+        @messages("importerAddress.hint")
+        <p/>
+    </h2>
+}
+
+@h4_1 = {
+    <strongr >
+        NE98 1AE
+    </strongr>
+}
+
+@h4_2 = {
+    <h2 class="govuk-hint">
+    @messages("importerAddressConfirmation.address")
+    </h2>
+}
+
+@main_template(
+    title = messages("importerAddressConfirmation.title")
+) {
+
+    @components.back_link()
+
+    @components.heading("importerAddressConfirmation.heading")
+
+    @pageHint
+
+    @h4_1
+    @link("", messages("importerAddressConfirmation.link1"))
+    <p/>
+    @h4_2
+    @govukSelect(Select(
```

Comment:
@somaraja I think we can just leave the DDLB empty and populate it when we have the data.

Reply:
@lmugnai As advised changes incorporated.